tests/Tests/700__sandbox/700__sandbox.robot

```diff
@@ -0,0 +1,115 @@
+*** Settings ***
+Documentation    Test Suite for Sandbox Test cases
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
+Library         String
+Library         SeleniumLibrary
+Suite Setup     Begin Web Test
+Suite Teardown  End Web Test
+
+
+*** Variables ***
+${SANDBOX_AUTH_TYPE}   *******
+${SANDBOX_USERNAME}    *******
+${SANDBOX_PASSWORD}    *******
+${SANDBOX_OCP_URL}     *******
+
+
+*** Test Cases ***
+Verify Application Switcher Have Only Opneshift Console Link
+    [Documentation]    Application switcher should have only
+    ...    openshift conosle link
+    [Tags]     ODS-309
+    ...        Sandbox
+    Check Openshift Console
+
+Verify ISV integration enablement IS Disabled
+    [Documentation]    All the isv integration
+    ...   component should be diabled
+    [Tags]     ODS-530
+    ...        Sandbox
+    Click Link    Explore
+    Capture Page Screenshot   explor.png
+    Wait Until Cards Are Loaded
+    Sleep    5s
+    ${link_elements}  Get WebElements    //article[@class="pf-c-card odh-card m-disabled"]
+    ${length}      Get Length    ${link_elements}
+    Run Keyword If  ${length} != ${11}    Fail     '${length}' tiles in Explore section is disabled
+
+Verify Support Link Doesn't Exist
+    [Documentation]   Only documentaion link is present
+    [Tags]    ODS-526
+    ...       Sandbox
+    Click Element    xpath=//*[@id="toggle-id"]
+    ${links_avialble}   Get WebElements
+    ...    //a[@class="odh-dashboard__external-link pf-c-dropdown__menu-item" and not(starts-with(@href, '#'))]
+    FOR  ${link}  IN  @{links_avialble}
+         ${href}    Get Element Attribute    ${link}    href
+         Run Keyword And Continue On Failure   Should Not Contain   ${href}   support   ignore_case=True
+    END
+
+Verify only small size is available for sandbox enviroinment
+    [Documentation]    Only Default and Small size is present
+    ...    on spawner
+    [Tags]     ODS-528
+    ...        Sandbox
+    Navigate To Page    Applications    Enabled
+    Launch JupyterHub Spawner From Dashboard
+    Match Small And Default Container Size
+
+Verify that idle Jupyterhub servers are culled
+    [Documentation]   Jupyterhub should be culled
+    ...    after 24 hours
+    [Tags]     ODS-547
+    Spawn Notebook With Arguments
+    ${jl_title}     Get Title
+    Switch Window    title=Red Hat OpenShift Data Science Dashboard
+    Sleep    24h
```

Comment:
shouldn't we change the timeout to a lower one? I would assume that if the timeout should work either cases

Reply:
for now i always verify after 24hr we could do that later however once culler is there we can modify it for now i will try to keep like this